x-pack/plugins/viz_editor/public/pseudo_plugins/pie_chart_plugin/pie_chart_plugin.tsx

```diff
@@ -162,14 +176,14 @@ function getSuggestionsForField(
       queries: {
         q1: {
           datasourceRef,
-          select: [{ ...firstOperation, alias: field.name }, { operator: 'count', alias: 'count' }],
+          select: [{ operator: 'count', alias: 'count' }, { ...firstOperation, alias: field.name }],
```

Comment:
Why have you reversed the order here?

Reply:
To match the order of xy chart which makes switching easier